stories/components/form/DropDown.stories.js

```diff
@@ -13,10 +13,10 @@ const options = [
 
 export const Basic = {
   args: {
-    eventType: 3,
+    value: 3,
     label: 'Test',
     options: options,
-    handleEventTypeChange: (e) => console.log(e.target.value),
+    onChange: (e) => console.log(e.target.value),
     className: ''
   }
 };
```

Comment:
The `DropDown.stories.js` file should be renamed to `Select.stories.js` to match the component